src/gui/properties/speedplotview.cpp

```diff
@@ -345,7 +345,7 @@ void SpeedPlotView::paintEvent(QPaintEvent *)
     rect.adjust(3, 0, 0, 0); // Need, else graphs cross left gridline
 
     const double yMultiplier = (niceScale.arg == 0.0) ? 0.0 : (static_cast<double>(rect.height()) / niceScale.sizeInBytes());
-    const double xTickSize = static_cast<double>(rect.width()) / m_viewablePointsCount;
+    const double xTickSize = static_cast<double>(rect.width()) / (m_viewablePointsCount - 1);
```

Comment:
We should be sure that `m_viewablePointsCount - 1` can't be 0.

Reply:
should I add an  `std::max` around it?

Reply:
well, It won't become 0 for now at least it stores the buffer sizes which are calculated at compile time.